tests/test-ansible-functional.sh

```diff
@@ -154,7 +154,9 @@ function enable_maas_api {
   #                  different virtualenv which are done in three tasks. These three tasks are being
   #                  run in this particular order to ensure "pyrax" installs correctly.
   PYTHON_BIN="$(which python)"
-  virtualenv --python="${PYTHON_BIN}" /opt/test-maas
+  virtualenv --python="${PYTHON_BIN}" --no-pip /opt/test-maas
+  curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```

Comment:
I don't see any issue w/ this change here, but curious why it's needed. Got more info?

Reply:
For some reason the pip inside virtualenv is causing error:

```unning virtualenv with interpreter /var/lib/jenkins/workspace/PR_rpc-maas-master-trusty-mitaka-deploy/rpc-maas/.tox/functional/bin/python
Using real prefix '/usr'
New python executable in /opt/test-maas/bin/python
Installing setuptools, pip, wheel...done.
Traceback (most recent call last):
  File "/opt/test-maas/bin/pip", line 7, in <module>
    from pip._internal import main
ImportError: No module named _internal```
```

We tried pretty hard but it's hard to reproduce.
By going over the internet seems like installing a fresh pip inside virutalenv would fix it, as it did.

Reply:
OK -- I remember hitting that error when we were testing stuff out manually. I can't remember which version of pip was automatically being installed in the venv though, I'd need to check my chat history.